KsApi/models/GraphUserCreditCardTests.swift

```diff
@@ -104,4 +104,41 @@ final class GraphUserCreditCardTests: XCTestCase {
       XCTFail("Failed to decode GraphUserCreditCard")
     }
   }
+
+  func testCreditCardsDecoding_unrecognizedCardType() {
+    let jsonString = """
+        {
+          "storedCards": {
+            "nodes": [
+                {
+                  "expirationDate": "2020-02-01",
+                  "lastFour": "1111",
+                  "type": "UNKNOWN",
+                  "id": "2768"
+                }
+            ],
+            "totalCount": 0
+          }
+        }
+        """
+
+    let data = jsonString.data(using: .utf8)
+
+    do {
+      //swiftlint:disable force_unwrapping
```

Comment:
Can we use `//swiflint:disable:next`? That should only disable the rule on the next line...otherwise it will disable it till end of file.